pyrato/parameters.py

```diff
@@ -202,3 +203,184 @@ def clarity(energy_decay_curve, early_time_limit=80):
+
+    # calculate lim1 - lim4 for each channel
+    lim1, lim2 = early_time_limit_sec, energy_decay_curve.times[-1]
+    lim3, lim4 = 0.0, early_time_limit_sec
+
+    # return in dB
+    return 10* np.log10(__energy_balance(lim1,
+                                         lim2,
+                                         lim3,
+                                         lim4,
+                                         energy_decay_curve,
+                                         energy_decay_curve))
+
+
+def __energy_balance(lim1, lim2, lim3, lim4,
+                     energy_decay_curve1,
+                     energy_decay_curve2):
+    r"""
+    Calculate the energy balance for the time limits from the two energy
+    decay curves (EDC). If second one is not provided, the first will be
+    used for both.
+
+    A collection of roomacoustic parameters are defined by their
+    time-respective energy balance, where the differentiation is made by
+    the four given time limits [#iso]_.
+
+    Energy-Balance is calculated as:
+
+    .. math::
+
+        EB(p) = 10 \log_{10} \frac{
+            \displaystyle \int_{lim3}^{lim4} p_2^2(t) \, dt
+        }{
+            \displaystyle \int_{lim1}^{lim2} p_1^2(t) \, dt
+        }
+
+    where :math:`lim1 - lim4` are the time limits and :math:`p(t)` is the
+    pressure of a room impulse response. Here, the energy balance is
+    efficiently computed from the EDC :math:`e(t)` directly by:
+
+    .. math::
+
+        EB(e) = 10 \log_{10} \left( \frac{e_2(lim3) -
+        e_2(lim4)}{e_1(lim1) - e_1(lim2)} \right).
+
+    Parameters
+    ----------
+    lim1, lim2, lim3, lim4 : float
+        Time limits (:math:`t_e`) in seconds.
+    energy_decay_curve1 : pyfar.TimeData
+        Energy decay curve 1 (EDC1) of the room impulse response
+        (time-domain signal). The EDC must start at time zero.
+    energy_decay_curve2 : pyfar.TimeData
+        Energy decay curve 2 (EDC2) of the room impulse response
+        (time-domain signal). The EDC must start at time zero.
+
+    Returns
+    -------
+    energy balance : numpy.ndarray[float]
+        energy-balance index (early-to-late energy ratio),
+        shaped according to the channel shape of the input EDC.
+
+    References
+    ----------
+    .. [#iso] ISO 3382, Acoustics — Measurement of the reverberation time of
+        rooms with reference to other acoustical parameters.
+    """
+
+    # Check input type
+    if not isinstance(energy_decay_curve1, pf.TimeData):
+        raise TypeError("energy_decay_curve1 must be a pyfar.TimeData object.")
+    if not isinstance(energy_decay_curve2, pf.TimeData):
+        raise TypeError("energy_decay_curve2 must be a pyfar.TimeData object.")
+
+    for name, val in zip(("lim1","lim2","lim3","lim4"), (lim1,
+                                                         lim2,
+                                                         lim3,
+                                                         lim4)):
```

Comment:
```suggestion
    for name, val in zip(
            ("lim1","lim2","lim3","lim4"), (lim1, lim2, lim3, lim4)):
```
Even though both versions are okay according to pep8, I'd vote for a new-line indent over a hanging indent here. It's much more space efficient and more readable if you ask me.